From 9555d6cfc33a58a7cf91d137f7209ccf1bee86a8 Mon Sep 17 00:00:00 2001
From: Lyonel Vincent <lyonel@ezix.org>
Date: Tue, 12 Oct 2021 22:02:22 +0200
Subject: [PATCH 56/65] code clean-up

get rid of warning complaining about `register` storage of CRC
---
 src/core/partitions.cc | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/core/partitions.cc b/src/core/partitions.cc
index 4268903..7b9fd4f 100644
--- a/src/core/partitions.cc
+++ b/src/core/partitions.cc
@@ -520,7 +520,6 @@ hwNode & partition)
  * - Now pass seed as an arg
  * - changed unsigned long to uint32_t, added #include<stdint.h>
  * - changed len to be an unsigned long
- * - changed crc32val to be a register
  * - License remains unchanged!  It's still GPL-compatable!
  */
 
@@ -626,7 +625,7 @@ uint32_t
 __efi_crc32(const void *buf, unsigned long len, uint32_t seed)
 {
   unsigned long i;
-  register uint32_t crc32val;
+  uint32_t crc32val;
   const unsigned char *s = (const unsigned char *)buf;
 
   crc32val = seed;
-- 
2.33.1

